.github/CODEOWNERS

```diff
@@ -39,7 +39,7 @@
 /core/templates/domain/ @kevintab95
```

Comment:
@vojtechjelinek Pleaes follow the instructions above (at the top of this file) for transferring codeowners. The per-line todos are important in case further changes are made to this file while you're away, and the issue is necessary for tracking.

Reply:
Done.